app/src/app/components/providers/ThemeProvider.tsx

```diff
@@ -1,10 +1,20 @@
 import { createContext, useContext, useEffect, useState } from "react";
 import { LocalStorageKeys } from "../../../lib/types/LocalStorageKeys";
 import { Theme, ThemeProviderProps, ThemeProviderState } from "./types/ThemeTypes";
+import { useGetBrandingConfig } from "./hooks/useGetBrandingConfig";
+import { ErrorComponent } from "../contents/common/ErrorComponent";
+
+// defaultTheme is to be used if both themes are available and no user preference is set in local storage
```

Comment:
constants should probably be capital seprated by unserscore... eg. `DEFAULT_THEME`... also can these constants and `getSystemsTheme` function be moved to another file to help reduce clutter in this file